Sources/swift-doc/Subcommands/Generate.swift

```diff
@@ -47,6 +47,15 @@ extension SwiftDoc {
     var options: Options
 
     func run() throws {
+      for directory in options.inputs {
+        var isDirectory: ObjCBool = false
+        if !FileManager.default.fileExists(atPath: directory, isDirectory: &isDirectory) {
+          logger.warning("Input directory \(directory) does not exist.")
```

Comment:
```suggestion
          logger.warning("Input path \(directory) does not exist.")
```

Small change to make message consistent with how we call input arguments "paths"